regression/benchmark/run_all.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+set -e
+
+# Ensure all dependencies are installed
+command -v jq >/dev/null 2>&1 || { echo >&2 "jq is not installed. Aborting."; exit 1; }
+
+# Script directory
+BENCHMARK_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+# Kernel image 
+KERNEL_DIR="/root/dependency"
+LINUX_KERNEL="${KERNEL_DIR}/vmlinuz"
+
+# Generate init script for Linux cases
+generate_init_script() {
+    local benchmark="$1"
+    local init_script=$(cat <<EOF
+#!/bin/sh
+
+echo "Running ${benchmark}"
+chmod +x /benchmark/${benchmark}/run.sh
+/benchmark/${benchmark}/run.sh
+
+poweroff -f
+EOF
+)
+    echo "$init_script"
+}
+
+run_benchmark() {
+    local benchmark="$1"
+    local avg_pattern="$2"
+    local avg_field="$3"
+
+    local linux_output="${BENCHMARK_DIR}/linux_output.txt"
+    local aster_output="${BENCHMARK_DIR}/aster_output.txt"
+    local result_template="${BENCHMARK_DIR}/${benchmark}/result_template.json"
+    local result_file="result_${benchmark}.json"
+
+    # Init script for initramfs
+    local init_script="${BENCHMARK_DIR}/init"
```

Comment:
The first process is called the init process. But the program that is run as the init process should not be called the init program. When it is a script, it should not be called "init script". 

```
# Init script for initramfs
```

Reading this comment, I would think this is a script that initializes the content of the initramfs. To avoid such ambiguity, describe what the script does in its name.

Reply:
I change it to `entrypoint` now. 
```sh
# Entrypoint script for initramfs
local initramfs_entrypoint_script="${BENCHMARK_DIR}/entrypoint.sh"
generate_entrypoint_script "${benchmark}" > "${initramfs_entrypoint_script}"
chmod +x "${initramfs_entrypoint_script}"
```